### PR TITLE
spelling, context & references

### PR DESCRIPTION
**Pull request by lindenstark**
I've made some minor spelling/grammatical changes and I've added references (**these should be expanded on**).

### CODE REVIEW

**Comment by McFranko** [comment]
Biblatex should be used for citations

**Comment by McFranko** [comment]
Actually, the references aren't really references, it would make more sense to have them in a glossary instead.

**Comment by McFranko** [comment]
Also, lines shouldn't go longer than 80 characters, and there are some line breaks in odd spots in the source

**Comment by lindenstark** [comment]
> Biblatex should be used for citations



> Actually, the references aren't really references, it would make more sense to have them in a glossary instead.

**Comment by McFranko** [comment]
Please use LaTeX's built in glossary feature. Also, the code sections are now only one line. THey are more readable when they are on multiple lines and follow Rust's style guide.

**Comment by McFranko** [comment]
https://www.overleaf.com/learn/latex/Glossaries

Here is an explanation of latex glossaries

File: whitepaper.tex [hunk 1]
```diff
@@ -153,109 +120,105 @@ \subsection{How PADLOCK Solves the Double Spend Problem}
...
+Due to transactions not being publicly disclosed on chain, it achieves a high
+level of privacy. DOS attacks can be countered by requiring proof of work to be
+submitted with each transaction. All this together creates a very robust base
+for a digital currency.
+
+\section{Glossary}
+
+\textbf{ASIC}: Application specific integrated circuit. 
+\newline{}
+\textbf{Segwit}: Freeing block size limit by removing signature data.
```
Comment (McFranko) [inline whitepaper.tex:219]: Segwit doesn't remove signature data, it makes it so it's not included in the block size cap.

Comment (McFranko) [reply whitepaper.tex:219]: Maybe say it's separates signature data instead

**Comment by McFranko** [comment]
I've added this to the linden-changes branch so I can work on it on my own and pull from there. Thanks so much for the help, I appreciate it

**Pull request closed without merging**